CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## [1.57.2](https://github.com/brave/brave-ios/releases/tag/v1.57.2)
+
+- Improved performance of closing all tabs on certain sites that cause many tabs to be open. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```

Comment:
How about something like this?

```
- Improved performance when closing all tabs opened due to "Block Popups" being disabled on certain websites. ([#8060](https://github.com/brave/brave-ios/issues/8060))
```

Reply:
Ok great. https://github.com/brave/brave-ios/commit/d71e90729b31dc2f49fdcd3dd345125079cb732e